extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketConnectorBase.java

```diff
@@ -123,6 +123,9 @@ Set<String> getPathParamNames(String path) {
     }
 
     String replacePathParameters(String path) {
+        if (path.isEmpty()) {
+            return "";
```

Comment:
I was going to add a comment to the processor but it's not really called often so I refrained to do it but in this case, I would simply return `path`.

Reply:
Well, `return "";` is compiled into bytecode that loads an entry from the constant pool. So the only difference is one more entry in the constant pool of `WebSocketConnectorBase` :shrug:.

Having said that, I don't know why I used this form here and I have no preference. So let's use `return path;` instead! ;-)

Reply:
Yeah I perfectly agree the cost is very small but small costs pile up and when there is absolutely no reason to pay this cost, I think we should avoid it.

Certainly not worth blocking the PR thus why I approved anyway and let you decide what to do :).

(I wouldn't have asked if it made the code worse or more complex)

Reply:
> but small costs pile up 

You would need a loooot of `return "";` statements to see a very tiny cost but I get your point and the PR is updated now ;-).

> Certainly not worth blocking the PR thus why I approved anyway and let you decide what to do :).
> (I wouldn't have asked if it made the code worse or more complex)

:100: